bangpy-ops/ops/celu/celu.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (C) [2021] by Cambricon, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+# CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+# pylint: disable=missing-docstring, invalid-name, too-many-locals
+"""A multi-platform code link example test for BANGPy TCP."""
+import bangpy
+from bangpy import tcp
+from bangpy.platform.bang_config import TARGET
+from bangpy.tcp.util import round_down
+from bangpy.tcp.runtime import TaskType
+
+DTYPES = [bangpy.float32]
+TARGET_LIST = ["mlu290"]
+KERNEL_NAME = "Celu"
+
+
+class Celu:
+    def __init__(self, dtype, target, task_num):
+        self.dtype = dtype
+        self.target = target
+        self.task_num = task_num
+        self.bp = tcp.TCP(target)
+        self.inplace = self.bp.Var("inplace")
+        self.length = self.bp.SizeVar("length")
```

Comment:
1. `=`号两侧使用空格隔开，135～136 行并为如此
2. 调用函数时入参变量使用空格隔开，118/119 行这样做的，其他处没有。

统一代码风格

Reply:
已修改

Reply:
未修改完全，请检查确认，如130～133
```suggestion
        f = self.bp.BuildBANG(
            inputs=[buffer_in0, buffer_alpha, self.inplace],
            outputs=[buffer_out],
            kernel_name=KERNEL_NAME, )
```

Reply:
已修改